spk/he853/src/installer.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# Package
+PACKAGE="he853"
+DNAME="HomeEasy HE853 USB device executable"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/${PACKAGE} /usr/local/bin/${PACKAGE}
```

Comment:
Ok my fault, I ask to remove the line ...
https://github.com/SynoCommunity/spksrc/pull/2672/commits/17cc84d578c383d31ed67b9dd880d2703a3ecd7a#diff-03d8f37cb986f32b80aa24a8662e5105R35